tests/text_secure_v3_tests.rs

```diff
@@ -34,6 +24,127 @@ fn dynamic_roundtrip_echo(){
     }
 }
 
+#[test]
+fn android_session_test () {
```

Comment:
The test vectors are taken from the LibAxolotl RatchetSessionTest to verify expected answers. I suppose a comment would be helpful